script/Ciel/Day 3/03_01_CIEL03_2_1.txt

```diff
@@ -131,19 +141,19 @@ Alright, I'll give you some of my sweet bread then!"
 [sha:b498dce3c4f37fe655b2ec128b59eb1705ca6e41]{
 -- Page 21, Offset 4906. Glued. Mods: @x.
 -- んじゃ遠慮なくいただきます」
-// it seems like いただきます isn't particularly consistently translated.
-// I'll leave this one for now.
  Well, I'll be digging in then."
 }
 [sha:e5fa6eae7bd3e1b0c2734eb305a8a9e09c225167]{
 -- Page 21, Offset 4907.
 -- 有彦は両手を合わせてから菓子パンを食べ始める。
-Arihiko clasps his hands together and begins to eat his bread.
+// Not exactly what he's doing here, but avoids the culture gap a bit
```

Comment:
I really don't think this localization is necessary. anime folk know what clapping your hands together means and I don't really like removing culture stuff like this if possible.